linux_os/guide/system/software/integrity/fips/grub2_enable_fips_mode/ansible/shared.yml

```diff
@@ -32,7 +32,6 @@
     state: present
   when:
     - aesni_supported.rc == 0
-    - ansible_distribution == 'RedHat'
```

Comment:
Since you are touching this rule, do you think you could try to add at least some basic test scenarios?